2022-11-25 10:38:49.819 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 10:40:09.563 [main.py] INFO    | 進入VG主頁
2022-11-25 10:40:11.791 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 10:40:12.526 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 10:40:12.625 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 10:40:19.625 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 10:40:26.727 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 10:40:29.718 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 10:40:42.785 [face_backend.py] INFO    | 人臉:註冊人臉資料 建檔成功
2022-11-25 11:14:47.202 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:14:59.454 [main.py] INFO    | 進入VG主頁
2022-11-25 11:15:01.282 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:15:02.067 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:15:02.118 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:15:05.543 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 11:15:23.816 [face_backend.py] WARNING | 人臉:註冊人臉資料 建檔失敗，該人名已經註冊過
2022-11-25 11:15:31.270 [face_backend.py] INFO    | 人臉:註冊人臉資料 建檔成功
2022-11-25 11:15:43.128 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 11:15:46.170 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:15:51.070 [face_api.py] INFO    | 執行人臉模型訓練
2022-11-25 11:15:51.144 [face_backend.py] INFO    | 人臉:人臉模型訓練subprocess啟動
2022-11-25 11:15:51.147 [face_backend.py] INFO    | 人臉:訓練中...
2022-11-25 11:25:18.548 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:25:43.356 [main.py] INFO    | 進入VG主頁
2022-11-25 11:25:45.076 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:25:45.882 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:25:45.954 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:25:50.051 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:25:53.681 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 11:26:03.346 [face_backend.py] INFO    | 人臉:註冊人臉資料 建檔成功
2022-11-25 11:26:04.886 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:26:10.079 [face_api.py] INFO    | 執行人臉模型訓練
2022-11-25 11:26:10.149 [face_backend.py] INFO    | 人臉:人臉模型訓練subprocess啟動
2022-11-25 11:26:10.152 [face_backend.py] INFO    | 人臉:訓練中...
2022-11-25 11:27:01.965 [face_backend.py] INFO    | 人臉:訓練完成
2022-11-25 11:27:04.508 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:27:09.218 [face_api.py] INFO    | 進出權限人員名單 更新成功
2022-11-25 11:27:10.739 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:27:26.301 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 11:27:58.132 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:29:19.021 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:32:02.007 [main.py] INFO    | 進入VG主頁
2022-11-25 11:32:04.097 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:32:04.889 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:32:04.949 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:32:13.515 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:32:15.334 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 11:36:49.682 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:48:04.276 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:48:11.911 [main.py] INFO    | 進入VG主頁
2022-11-25 11:48:29.420 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:52:35.644 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:20.357 [main.py] INFO    | 進入VG主頁
2022-11-25 11:53:24.986 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:53:25.713 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:25.766 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:53:33.625 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:40.105 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:46.402 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:52.950 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:53:59.468 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:54:05.843 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:54:12.420 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:54:18.843 [face_detector.py] INFO    | 人臉:沒找到已訓練過的人臉，創建空白的list
2022-11-25 11:54:23.751 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:54:25.186 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:54:27.049 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 11:54:52.703 [face_backend.py] INFO    | 人臉:註冊人臉資料 建檔成功
2022-11-25 11:54:54.282 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:54:57.543 [face_api.py] INFO    | 執行人臉模型訓練
2022-11-25 11:54:57.609 [face_backend.py] INFO    | 人臉:人臉模型訓練subprocess啟動
2022-11-25 11:54:57.613 [face_backend.py] INFO    | 人臉:訓練中...
2022-11-25 11:55:53.646 [face_backend.py] INFO    | 人臉:訓練完成
2022-11-25 11:55:56.347 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 11:55:58.892 [face_api.py] INFO    | 進出權限人員名單 更新成功
2022-11-25 11:56:00.335 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 11:56:31.330 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 11:56:51.540 [main.py] INFO    | 有正在執行的子系統，嘗試關閉中...
2022-11-25 11:56:51.543 [main.py] ERROR   | main:terminate current subprocess failed, 'Thread' object has no attribute 'terminate'
2022-11-25 11:56:51.586 [main.py] INFO    | 進入VG主頁
2022-11-25 11:56:52.274 [main.py] INFO    | 進入VG主頁
2022-11-25 11:56:54.572 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 11:56:55.865 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 12:01:48.412 [main.py] INFO    | 進入VG主頁
2022-11-25 12:01:50.833 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 12:01:51.963 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 12:02:54.464 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:12:00.092 [main.py] INFO    | 進入VG主頁
2022-11-25 13:12:02.731 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 13:12:03.695 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:12:14.648 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 13:12:21.454 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:13:05.159 [main.py] INFO    | 進入VG主頁
2022-11-25 13:13:07.777 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 13:13:08.723 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:13:37.238 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:13:49.952 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:13:59.473 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:14:44.172 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:14:55.572 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:15:04.348 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:15:57.468 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:19:53.450 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 13:19:53.452 [face_backend.py] INFO    | 人臉:收到[CLICK_CHANGE_SETTING]的signal
2022-11-25 13:20:08.135 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:13.139 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:18.145 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:23.151 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:28.156 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:30.335 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 13:20:33.163 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:38.167 [face_backend.py] INFO    | 人臉:收到[CLICK_CHANGE_SETTING]的signal
2022-11-25 13:20:42.466 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:47.471 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:20:47.654 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 13:20:52.475 [face_backend.py] INFO    | 人臉:收到[CLICK_CHANGE_SETTING]的signal
2022-11-25 13:23:37.600 [face_api.py] INFO    | 辨識登入門檻值設定變更成功
2022-11-25 13:23:37.601 [face_backend.py] INFO    | 人臉:收到[CLICK_CHANGE_SETTING]的signal
2022-11-25 13:23:58.465 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:24:15.771 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:24:20.775 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:45:35.893 [main.py] INFO    | 進入VG主頁
2022-11-25 13:45:38.838 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 13:45:39.911 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:45:56.445 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:46:09.316 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:46:21.256 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:46:32.747 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:46:38.577 [face_backend.py] INFO    | 人臉:成功儲存新設定的圍籬區域:
[[(482, 60), (1217, 52), (1140, 727), (486, 745)]]
2022-11-25 13:46:38.580 [face_api.py] INFO    | 虛擬圍籬設定成功
2022-11-25 13:46:38.621 [face_backend.py] INFO    | 人臉:收到[UPDATE_FENCE]的signal
2022-11-25 13:47:07.601 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:47:16.865 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:47:27.947 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:47:40.235 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:47:51.973 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:48:00.500 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:51:36.091 [face_backend.py] INFO    | 人臉:成功儲存新設定的圍籬區域:
[[(482, 60), (1217, 52), (1140, 727), (486, 745)]]
2022-11-25 13:51:36.097 [face_api.py] INFO    | 虛擬圍籬設定成功
2022-11-25 13:51:36.140 [face_backend.py] INFO    | 人臉:收到[UPDATE_FENCE]的signal
2022-11-25 13:55:53.908 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:55:58.198 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 13:56:04.313 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 13:56:07.432 [face_api.py] INFO    | 已訓練的人臉照片資料 刪除成功
2022-11-25 13:56:07.436 [face_backend.py] INFO    | 人臉:收到[RELOAD_FRATURE]的signal
2022-11-25 13:56:09.360 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 13:56:11.716 [face_api.py] INFO    | 進入人臉註冊頁
2022-11-25 13:56:52.964 [face_backend.py] INFO    | 人臉:註冊人臉資料 建檔成功
2022-11-25 13:56:55.430 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 13:56:58.960 [face_api.py] INFO    | 執行人臉模型訓練
2022-11-25 13:56:59.129 [face_backend.py] INFO    | 人臉:人臉模型訓練subprocess啟動
2022-11-25 13:56:59.133 [face_backend.py] INFO    | 人臉:訓練中...
2022-11-25 13:59:02.722 [face_backend.py] INFO    | 人臉:訓練完成
2022-11-25 13:59:02.764 [face_backend.py] INFO    | 人臉:收到[RELOAD_FRATURE]的signal
2022-11-25 13:59:12.015 [face_api.py] INFO    | 進入人臉模型訓練頁
2022-11-25 13:59:15.717 [face_api.py] INFO    | 進出權限人員名單 更新成功
2022-11-25 13:59:19.906 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 13:59:31.008 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 13:59:53.472 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 14:00:15.834 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 14:00:39.124 [face_backend.py] INFO    | 人臉:收到[UNLOCK_DOOR]的signal
2022-11-25 15:43:14.236 [main.py] INFO    | 進入VG主頁
2022-11-25 15:43:21.943 [face_backend.py] INFO    | 人臉: subprocess啟動
2022-11-25 15:43:22.882 [face_api.py] INFO    | 進入人臉管制(UI)登入頁
2022-11-25 15:43:57.485 [main.py] INFO    | 有正在執行的子系統，嘗試關閉中...
2022-11-25 15:43:57.486 [main.py] ERROR   | main:terminate current subprocess failed, 'Thread' object has no attribute 'terminate'
2022-11-25 15:43:57.515 [main.py] INFO    | 進入VG主頁
2022-11-25 15:43:58.102 [main.py] INFO    | 進入VG主頁
2022-11-25 15:44:00.329 [wearing_api.py] INFO    | 穿戴: 進入辨識頁
2022-11-25 15:55:24.812 [main.py] INFO    | 進入VG主頁
2022-11-25 15:55:27.006 [vf_api.py] INFO    | 測距/物件: subprocess啟動
2022-11-25 15:55:38.189 [vf_api.py] INFO    | 測距/物件:進入測距監控系統
2022-11-25 15:56:04.989 [vf_backend.py] INFO    | 測距/物件:收到[CHANGE_DETECT_MODE]的signal
2022-11-25 17:01:06.876 [main.py] INFO    | 進入VG主頁
2022-11-25 17:01:10.980 [wearing_api.py] INFO    | 穿戴: 進入辨識頁
2022-11-25 17:02:40.176 [main.py] INFO    | 進入VG主頁
2022-11-25 17:02:42.532 [wearing_api.py] INFO    | 穿戴: 進入辨識頁
2022-11-25 17:04:26.714 [main.py] INFO    | 進入VG主頁
2022-11-25 17:04:28.696 [wearing_api.py] INFO    | 穿戴: 進入辨識頁
2022-11-25 17:07:45.043 [main.py] INFO    | 進入VG主頁
2022-11-25 17:07:48.393 [wearing_api.py] INFO    | 穿戴: subprocess啟動
